stdlib/inspect.pyi

```diff
@@ -421,10 +421,9 @@ class BoundArguments:
 # Classes and functions
 #
 
-# TODO: The actual return type should be list[_ClassTreeItem] but mypy doesn't
-# seem to be supporting this at the moment:
-# _ClassTreeItem = list[_ClassTreeItem] | Tuple[type, Tuple[type, ...]]
-def getclasstree(classes: list[type], unique: bool = False) -> list[Any]: ...
+_ClassTreeItem: TypeAlias = list[_ClassTreeItem] | tuple[type, tuple[type, ...]]
```

Comment:
I don't think that it ever returns a tuple. See: 
- `walktree` which is called in `getclasstree` https://github.com/python/typeshed/blob/693add9512ac0db5bb877e1e57fd0e46fb22158e/stdlib/inspect.pyi#L428
- source of `walktree` https://github.com/python/cpython/blob/71ede1142ddad2d31cc966b8fe4a5aff664f4d53/Lib/inspect.py#L1158-L1164

But, the list contains `tuple` items, see https://github.com/python/cpython/blob/71ede1142ddad2d31cc966b8fe4a5aff664f4d53/Lib/inspect.py#L1161